### PR TITLE
navigate() and setReportsFilters() methods in Fullscreen widgets

### PR DESCRIPTION
**Pull request by bolchowka**
This PR adds two methods for Fullscreen widgets in `agent-app-sdk`:
* `navigate` - allows to navigate the Agent App to a given path
* `setReportsFilters` - allows to modify filters in the Reports section so that navigation to this section works properly

### CODE REVIEW

**Comment by sgraczyk** [comment]
@bolchowka Can you confirm that all changes in [package-lock.json](https://github.com/livechat/products-sdk/pull/26/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) are expected?

File: packages/agent-app-sdk/README.md [hunk 1]
```diff
@@ -247,6 +247,14 @@ This widget currently does not support any events.
 
 Displays a red badge on top of the Fullscreen app icon. Use this to notify Agents there’s something important inside the widget. Make sure Agents can dismiss the notification to avoid cluttered UI.
 
+#### `navigate(pathname: string): Promise<void>`
+
+Navigates LiveChat Agent App to given pathname.
+
+#### `setReportsFilters(filters: any): Promise<void>`
```
Comment (sgraczyk) [inline packages/agent-app-sdk/README.md:254]: Could we use dedicated types? `any` seems very "unhelpful"

**Review by sgraczyk — APPROVED** [review]
The change looks fine, however, I believe we should define types for filters in reports

**Review by jakubsikora — APPROVED** [review]

File: packages/agent-app-sdk/README.md (hunk 1 above)
Comment (bolchowka) [reply packages/agent-app-sdk/README.md]: Fixed in [5e45d12](https://github.com/livechat/products-sdk/pull/26/commits/5e45d128e28719ff10b82722c6d8f978430d9146)

**Comment by bolchowka** [comment]
> @bolchowka Can you confirm that all changes in [package-lock.json](https://github.com/livechat/products-sdk/pull/26/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) are expected?

@sgraczyk yes.

This PR removes `yarn.lock` as the project uses `npm` rather than `yarn` (see [CONTRIBUTING.md](/livechat/products-sdk/blob/master/CONTRIBUTING.md) instructions). For some reason, `package-lock.json` has not been updated for a long time.

**Pull request merged**